### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by amantulsyan35**
Homepage for quiz app

### CODE REVIEW

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge.

File: Homepage/index.html [hunk 1]
```diff
@@ -41,11 +41,57 @@
...
       <div class="homepage-quiz-container">
-        <div class="quiz-category-card">x</div>
-        <div class="quiz-category-card">x</div>
-        <div class="quiz-category-card">x</div>
-        <div class="quiz-category-card">x</div>
-        <div class="quiz-category-card">x</div>
+        <div class="card">
+          <div class="card-hero">
+            <img
+              src="https://www.sbs.com.au/popasia/sites/sbs.com.au.popasia/files/styles/full/public/naruto_manga_feat.jpg?itok=gei9zHwJ&mtime=1513570505"
```
Comment (mehulsatardekar) [inline Homepage/index.html:47]: add an alt text for the image description even if you image wont load atleast user can see the text

File: Homepage/index.html [hunk 2]
```diff
@@ -41,11 +41,57 @@
...
+              information. I am convenient because I require little markup to
+              use effectively.
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-hero">
+            <img
+              src="https://static1.cbrimages.com/wordpress/wp-content/uploads/2021/04/attack-on-titan-manga-should-continue-stop.png?q=50&fit=crop&w=960&h=500&dpr=1.5"
+              alt=""
```
Comment (mehulsatardekar) [inline Homepage/index.html:65]: do the same changes here

**Review by mehulsatardekar — APPROVED** [review]
@amantulsyan35 Just a suggestion on UI use a little bit of grayish color for the body, so the card container would look even better. you can totally skip this it's just a suggestion. other than that good to merge this branch

File: Homepage/index.html (hunk 1 above)
Comment (amantulsyan35) [reply Homepage/index.html:47]: surely will do that

**Pull request merged**